.agents/skills/release-notes/scripts/verify.py

```
@@ -0,0 +1,57 @@
+    urls = extract_urls(content)
+    if not urls:
+        print("No URLs found in input.")
+        sys.exit(1)
+
+    errors = []
+    for url in urls:
+        if not url.startswith("https://"):
+            errors.append(f"NOT HTTPS: {url}")
+            continue
+        try:
+            req = urllib.request.Request(url, method="HEAD", headers={"User-Agent": "url-verify/1.0"})
```

Ruff (0.15.20)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)